catalog/USGS/NLCD_RELEASES/USGS_RCMAP_RELEASES_V5_COMPONENTS.jsonnet

```diff
@@ -0,0 +1,358 @@
+
+    Herbaceous is a continuous field component consisting of grasses, forbs and cacti which were photosynthetically active at any point in the year of mapping.
+
+    Non-sagebrush shrub is a continuous field component encompassing all shrub species not of the sagebrush (Artemisia spp.) genus. Shrubs, in general, are discriminated by the presence of woody stems and &lt; 6-m in height.
+
+    Perennial herbaceous is a continuous field component consisting of grasses, forbs and cacti which were photosynthetically active at any point in the year of mapping and whose lifecycle includes more than one growing season (includes biennials).
+
+    Litter is a continuous field component including dead standing woody vegetation, detached plant organic matter and biological soil crusts.
+
+    Sagebrush is a continuous field component encompassing almost all species of Sagebrush (Artemisia spp.) including Big Sagebrush (A. tridentata spp.), Low Sagebrush (A. arbuscula), Black Sagebrush (A. nova), Three-tip Sagebrush (A. triparta) and Silver Sagebrush (A. cana). This component is nested within Shrub as a secondary component. Excludes the low stature prairie sage (A. frigida) and white sagebrush (A. ludoviciana).
+
+    Shrub is a continuous field component encompassing all shrub species discriminated by the presence of woody stems and &lt; 6-m in height.
+
+    Tree cover is defined as vegetation with persistent woody stems &gt; 6m in height. Mature stand of pinyon (Pinus spp.  and juniper (Juniperus spp.) are included regardless of height.
+
+    See also:
+    Hua Shi, Matthew Rigge, Kory Postma & Brett Bunde (2022) Trends analysis of rangeland condition monitoring assessment and projection (RCMAP) fractional component time series (1985–2020), GIScience & Remote Sensing, 59:1, 1243-1265, DOI: 10.1080/15481603.2022.2104786
+
+    Rigge, M., H. Shi, C. Homer, P. Danielson, and B. Granneman. 2019. Long-term trajectories of fractional component change in the Northern Great Basin, USA. Ecosphere: e02762.
+
+     Rigge, M., C. Homer, L. Cleeves, D. Meyer, B. Bunde, H. Shi, G. Xian, and M. Bobo. 2020. Quantifying western U.S. rangelands as fractional components with multi-resolution remote sensing and in situ data. Remote Sensing. 12: 412.
+
+     Rigge, M., Homer, C., Shi, H., Meyer, D., Bunde, B., Granneman, B., Postma, K., Danielson, P., Case, A., and Xian, G. 2021. Rangeland Fractional Components Across the Western United States from 1985 to 2018. Remote Sensing, 13: 813.
+|||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+  ],
+  keywords: [
+    'shrublands',
```

Comment:
keywords must be sorted and not include spaces

Reply:
Updated to alphabetical order. Not sure on the spaces. please check that again and let me know more specifics.